submissions/maryOkosun/HTML/product.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>PhoneHub</title>
+</head>
+<body>
+	<header>
+		<div>
+			<h1>Phone Hub</h1>
+		</div>
+		<div>
+		<nav>
+			<a href="product.html"><li>Home </li></a>
+			<a href="#"><li>Login </li></a>
+			<a href="#"><li>Help </li></a>
+			<a href="#"><li>Cart</li></a>
+		</nav>
+		</div>
+		<ol>
+		<li>
+		 	<img src="images/image1.png" alt="phone">
+		 	
+		 	<h2>TECNO SPARK </h2>
+		 	<h3>PRICE: <em>&#x20A6;36,990</em> </h3>
+		 	<h2>DESCRIPTION</h2>
+		 	
+			 	It has an Android 7.0 (Nougat) OS and a Screen Size of	5.7 inches. A Display Resolution of	1440 x 720 pixels and a Processor of Quad-core MediaTek MT6737. Has a RAM of 2GB and a Storage Space of	16GB. Also an Expansion Slot of	128GB and the Main Camera	is 13 megapixels. The Selfie Camera is	13 megapixels and Battery	of 3000mAh.
+			
+	
+		</li>
+		<hr>
+		<li>
+		 	<img src="images/image2.jpeg" alt="phone">
+		 	
+		 	<h2>SAMSUNG GALAXY S9 </h2>
+		 	<h3>PRICE:<em>&#x20A6;105,000</em> </h3>
+		 	<h2>DESCRIPTION</h2>
+		 	
+			 	It has an Android 7.0 (Nougat) OS and a Screen Size of	5.7 inches. A Display Resolution of	1440 x 720 pixels and a Processor of Quad-core MediaTek MT6737. Has a RAM of 2GB and a Storage Space of	16GB. Also an Expansion Slot of	128GB and the Main Camera	is 13 megapixels. The Selfie Camera is	13 megapixels and Battery	of 3000mAh.
+			
+		 		
+		</li>
+		<hr>
+		<li>
+		 	<img src="images/image3.png" alt="phone">
+		 	
+		 	<h2>TECNO CAMON CM </h2>
+		 	<h3>PRICE:<em>&#x20A6;45,000</em> </h3>
+		 	<h2>DESCRIPTION</h2>
+		 	
+			 	It has an Android 7.0 (Nougat) OS and a Screen Size of	5.7 inches. A Display Resolution of	1440 x 720 pixels and a Processor of Quad-core MediaTek MT6737. Has a RAM of 2GB and a Storage Space of	16GB. Also an Expansion Slot of	128GB and the Main Camera	is 13 megapixels. The Selfie Camera is	13 megapixels and Battery	of 3000mAh.
+				
+		</li>
+		</ol>
+		<hr>
+		
+		<footer>
+			<h4>CONTACT </h4>
+			<p>
+		<ul>
+			<li><a href="https://www.twitter.com">Twitter</a> </li>
+			<li><a href="https://www.linkedIn.com">LinkedIn</a></li>
+			<li><a href="https://www.medium.com">Medium</a></li>
+		</ul>
+	</p>
+	<hr>
```

Comment:
I don't think you need 2 of the horizontal line. One is just enough @Marienoir

Reply:
I have implemented the changes. Thanks for the correction.